exchanges/exchange.go

```diff
@@ -575,8 +579,14 @@ func (e *Base) SetCurrencyPairFormat() error {
 
 // GetAuthenticatedAPISupport returns whether the exchange supports
 // authenticated API requests
-func (e *Base) GetAuthenticatedAPISupport() bool {
-	return e.AuthenticatedAPISupport
+func (e *Base) GetAuthenticatedAPISupport(endpoint uint8) bool {
```

Comment:
To me a bool isn't explicit in interpreting what the code is doing. I don't want to see `GetAuthenticatedAPISupport(false)` anywhere since it doesn't really help knowing whats happening at a glance. Having `GetAuthenticatedAPISupport(exchange.RestAuthentication)` says what is the desired outcome while also allowing for more definitions in future.

If you'd like I could change it to have predefined bool values so that it is still explicit but only uses a bool instead of a uint

Reply:
I see your point, thanks for clarifying. I don't have an issue with it, its only 1 byte either way.